parsec/core/fs/workspacefs/remanence_manager.py

```diff
@@ -196,9 +197,11 @@ def manage_events(self) -> Iterator[None]:
             with self.event_bus.connect_in_context(
                 (CoreEvent.FS_ENTRY_DOWNSYNCED, cast(EventCallback, self._on_entry_synced)),
                 (CoreEvent.FS_ENTRY_SYNCED, cast(EventCallback, self._on_entry_synced)),
-                (CoreEvent.FS_BLOCK_DOWNLOADED, cast(EventCallback, self._on_block_downloaded)),
-                (CoreEvent.FS_BLOCK_PURGED, cast(EventCallback, self._on_block_removed)),
             ):
+                self.fs_block_event_bus.connect_downloaded(
```

Comment:
when those are disconnected ? (this is why we use connect_in_context context manager)

Reply:
Why do you want to disconnect ? :thinking:

Reply:
Done